docs/FAQ.md

```diff
@@ -109,6 +109,17 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### What do I do with divergent changes?
```

Comment:
I would also mention divergent branches here, as they also belong in the same bucket. Here's my suggestion:
> What do I do with divergent changes and branches?

Reply:
Speaking of that, should we start calling them "conflicted changes" instead "divergent changes"? We call branches "conflicted", not "divergent". We also use "conflicted" for files. I don't think we need to change the terminology before this PR gets merged.

Reply:
I think we should call them "conflicted changes" for user-facing documentation, while keeping "divergence" for technical documentation.

Reply:
I think that if we use "conflicted", it should be "conflicted change ids", since it's unclear what a "change" is and whether it's different from "revision" (in my mind, these are the same or very close). I use "divergent changes" to refer to "revisions with conflicted change ids".

Reply:
So, I'd title this:

> How do I deal with conflicted change ids (?? after the id), AKA divergent revisions?

Reply:
I don't mind changing to conflicted changes / change IDs if there's consensus on it but personally, as a user, they feel like two different things to me, and using the same "name" to describe them would make it more confusing for me.

Reply:
I think it makes sense to use the same name because branches and change ids behave quite similarly, but I don't want that discussion to block this PR. If you put the question about conflicted branches in a separate question, then the different terminology becomes less apparent.

Reply:
OK, I updated the PR with a separate question for branches.

I agree about branches and change IDs using the same terminology[1], but I still prefer divergent (or some other term that isn't conflict) to represent that, since to me "conflicted change/branch" makes me think it's a change/commit that has a file conflict in it. But as you say, that's a separate discussion :) 

[1] Though I would argue that they should also be represented the same way in the log